modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -22,17 +22,16 @@
 	alt_titles = list("Salon Manager", "Salon Technician", "Stylist", "Colorist")
 
 /datum/job/bartender
-	alt_titles = list("Bartender", "Mixologist")
+	alt_titles = list("Bartender", "Mixologist", "Barkeeper")
 
 /datum/job/blueshield
 	alt_titles = list("Blueshield", "Command Bodyguard", "Executive Protection Agent")
 
-
 /datum/job/botanist
 	alt_titles = list("Botanist", "Hydroponicist", "Gardener", "Botanical Researcher", "Herbalist")
 
 /datum/job/bouncer
-
+	alt_titles = list("Bouncer", "Service Guard") //was originally Doorman, Door Supervisor here, goof request
```

Comment:
```suggestion
	alt_titles = list("Bouncer", "Service Guard")
```
Those comments really dont need to be there; same goes for all other department guards

Reply:
they aint doing no harm

Reply:
I mean
They say "Originally", which implies that we changed an old value. That's not the case, they were debated changes that were scrapped.